crates/shared/src/sources/mod.rs

```diff
@@ -68,6 +68,7 @@ pub fn defaults_for_network(chain: &Chain) -> Vec<BaselineSource> {
             BaselineSource::UniswapV3,
         ],
         Chain::Sepolia => vec![BaselineSource::TestnetUniswapV2],
+        Chain::Hardhat => vec![BaselineSource::TestnetUniswapV2],
```

Comment:
This should be empty, no? Test nodes don't deploy any liquidity on their own so there is no default liquidity we could use here.

Reply:
Adjusted.